app/extensions/brave/locales/bn-BD/app.properties

```diff
@@ -0,0 +1,111 @@
+reload=Reload
```

Comment:
It looks like bn-BD properties files is loading in english strings.  Is that expected?